deploy/templates/toolchaincluster/host-role.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: toolchaincluster-host
+  namespace: toolchain-host-operator
```

Comment:
The namespace shouldn't be hardcoded - it should be set based on where the host operator is running.

Reply:
good point. Done https://github.com/codeready-toolchain/host-operator/pull/974/commits/7965e579ad772c35f6cbca874055e15d329729d6 Thanks!

Reply:
cool, but couldn't this be set via a go-template variable? wouldn't it be a bit more intuitive?

Reply:
Makes sense, go-template variables are now supported. For now just `.Namespace`.